### DISCUSSION TITLE
Total disappointment

### DISCUSSION THREAD

**Discussion by Anixx** [lencx/ChatGPT#1440] [General]
So, I installed the ChatGPT app from the MS Stgore, but it is awfully disappointing. It is an Electron app, which means that

* It brings with itself the whole standalone Chromium browser, and takes 286 MB of drive space (even though its main functionality is to serve as a chat window and nothing more)

* It does not follow the system theme style, except the dark/light mode.

* It does not use the system fonts: neither sybpixel smoothing, nor the font family, nor the color. It uses grey font on
    white background, which coupled with greyscale antialiasing makes the font unreadable (as opposed to my system theme which uses 

* Everything is gigantic and uses a lot of screen space, which is precise on a laptop. It looks like it is designed for a phone with touch interface.

**Comment by endvater** [comment]
Sehr gutes Timing für diesen RFC — der RegulatoryParser aus PR #11 liefert genau die strukturelle Grundlage, die Per-Claim Provenance braucht. Ich möchte eine Implementierungsfrage einwerfen, die meiner Meinung nach entscheidend ist:

## Die Kernfrage: Claim-Level oder Chunk-Level?

Es gibt zwei fundamental unterschiedliche Granularitätsstufen:

**Option 1 — Chunk-Level Provenance (machbar heute)**
Jeder `Befund` zeigt, aus welchen Chunks (`regulatory_reference` aus dem RegulatoryParser) die Antwort zusammengesetzt wurde:
```json
"quellen": [
  { "regulatory_reference": "§ 10 Abs. 3 GwG", "source": "gwg_2023.pdf", "retrieval_score": 0.91 },
  { "regulatory_reference": "Modul AT 7.3, Tz. 1", "source": "marisk_2022.pdf", "retrieval_score": 0.84 }
]
```
Das ist mit dem aktuellen Stack (RegulatoryParser + LlamaIndex Metadaten) direkt umsetzbar.

**Option 2 — Sentence-Level Provenance (sehr aufwändig)**
Jede einzelne Aussage im `begruendung`-Text wird einem spezifischen Quellsatz zugeordnet. Das braucht entweder NLI (Natural Language Inference) oder explizites Chain-of-Thought im Prompt.

---

## Mein Votum: Chunk-Level als v1, mit Pfad zu Sentence-Level

Für einen ersten PR würde ich strikt auf Chunk-Level bleiben und dafür sorgen, dass die `regulatory_reference`-Metadaten aus PR #11 zuverlässig in den `Befund` wandern. Das liefert bereits massiven Mehrwert für Wirtschaftsprüfer.

Sentence-Level Provenance kann v2 sein — gesteuert durch ein `--deep-provenance`-Flag, das den LLM-Prompt erweitert und die Antwortzeit entsprechend erhöht.

**Konkrete Frage an den RFC-Autor:** Ist Chunk-Level Provenance bereits abgedeckt, oder zielt ihr direkt auf Sentence-Level? Das würde meinen Implementierungsaufwand-Kommentar stark verändern.
